### PR TITLE
Additions based on Amphora notes

### PR DESCRIPTION
**Pull request by timbeiko**

### CODE REVIEW

File: Merge/mainnet-readiness.md [hunk 1]
```diff
@@ -92,15 +92,22 @@ This document outlines various tasks to work through to make the Merge ready for
 
 * [ ] Transition process tests with fully enabled consensus and execution layer
     * [ ] Hive: scenario with PoW network partitioning
-    * [ ] Hive: happy case and various edge case scenarios, e.g. re-org beyond transition block, etc
+    * [ ] Hive: happy case and various edge case scenarios, e.g:
+      * [ ] Re-org beyond transition block;
```
Comment (djrtwo) [inline Merge/mainnet-readiness.md:96]: No need for semicolons or periods in the markdown list, imo

File: Merge/mainnet-readiness.md [hunk 2]
```diff
@@ -127,6 +134,9 @@ Most research has been completed. Only listing things still left to dig deeper i
     * [ ] Sync during transition period
 * [ ] Discovery [is there actually anything to do here?]
 * [ ] Execution-layer proof of custody
+* [ ] Disaster recovery if invalid chain finalized 
+  * [x] [WIP] Client multiplexers ([link](https://github.com/karalabe/minority))
```
Comment (djrtwo) [inline Merge/mainnet-readiness.md:138]: note, this doesn't help with the disaster recovery wrt consensus/validators. if validators use it, it might avoid the disaster in the first place

File: Merge/mainnet-readiness.md [hunk 3]
```diff
@@ -127,6 +134,9 @@ Most research has been completed. Only listing things still left to dig deeper i
...
 * [ ] Further threat analysis
     * [ ] Miner attacks
     * [ ] Resource exhaustion post-merge
+* [ ] Fee Market behavior changes (missed slots impact)
```
Comment (timbeiko) [inline Merge/mainnet-readiness.md:142]: I'm not sure this is relevant for the merge itself,.

Comment (djrtwo) [reply Merge/mainnet-readiness.md:142]: I'd like to see an EIP specifying this soon, knowing that it likely won't go into the merge

Comment (timbeiko) [reply Merge/mainnet-readiness.md:142]: cc: @adietrichs :-)
